### ISSUE TITLE
RTMSUBST missing from repository

### ISSUE THREAD

**Issue by robert-j** [Baron-von-Riedesel/HX#42]
RTMSUBST for the 16-bit Borland Powerpack extender is missing from this repository. The last version I've found was part of[ HXD16217.zip](https://www.japheth.de/Download/HX/HXD16217.zip).

It's not that important for me, but I have a couple of patches for RTM and USER, for which I would send push requests.

Thanks for still supporting HX after so many years :)

**Comment by robert-j** [comment]
RTMSUBST has replaced Borland Powerpack's own 16-bit extender (RTM.EXE). It was indeed incomplete and buggy, but easy to fix.

- HX RTM.DLL implements the non-windows parts of RTM.EXE (only 3 functions, IIRC)
- HX USER.DLL implements a couple of Windows USER functions provided by RTM.EXE but not by DPMILD16, which only provides KERNEL16.

There are 3 versions of Powerpack in the wild: from Borland C++ 3.x, from Borland Pascal 7 and lastly the one of Delphi 1. The latest version is 1.5 from Delphi (and TASM 5, IIRC).

Version 1.5 is backwards compatible, and I've extended RTMSUBST to support it in full.

**Comment by Baron-von-Riedesel** [comment]
And what is the benefit of those fixes? Are there applications that won't work anymore with the Borland extender but work now with dpmild16/rtm.dll/user.dll?

**Comment by robert-j** [comment]
Good point. There are no real world benefits because a resident HX DPMI server is already good enough.

RTMSUBST is only needed when the NE stub is replaced to avoid loading the DPMI server in advance.

**Comment by Baron-von-Riedesel** [comment]
What would be cool is if TD.EXE runs flawless with debugging protected-mode apps (16stub). It works, but at debugger exit there's a crash, requiring a reboot. Perhaps there's a fix...
